paddle/framework/framework.proto

```diff
@@ -118,6 +118,38 @@ message LoDTensorArrayDesc {
 
 message ReaderDesc { repeated LoDTensorDesc lod_tensor = 1; }
 
+message MetaDataDesc {
+  enum DataType {
+    BOOL = 1;
+    INT16 = 2;
+    INT32 = 3;
+    INT64 = 4;
+    FP16 = 5;
+    FP32 = 6;
+    FP64 = 7;
+    TENSOR = 8;
+    LOD_TENSOR = 9;
+    SELECTED_ROWS = 10;
+    LOD_TENSOR_ARRAY = 11;
```

Comment:
I am not sure how we will be able to support channel of type channel. Go allows this. Do you think this code would suffice for that? Also, I am not sure if we need it right now.